packages/ove-app-networks/src/client/common/networks.js

```diff
@@ -13,6 +13,235 @@ $(function () {
+                return filter.value;
+            case Constants.Evaluation.FUNCTION_CALL:
+                return evaluateF(element, filter.func, filter.args);
+            case Constants.Evaluation.EQUALS:
+                // We don't want to force type comparisons in this case.
+                return evaluate(element, filter.left) == evaluate(element, filter.right); // eslint-disable-line
+            case Constants.Evaluation.NOT_EQUALS:
+                // We don't want to force type comparisons in this case.
+                return evaluate(element, filter.left) != evaluate(element, filter.right); // eslint-disable-line
+            case Constants.Evaluation.LESS_THAN:
+                return evaluateN(element, filter.left) < evaluateN(element, filter.right);
+            case Constants.Evaluation.GREATER_THAN:
+                return evaluateN(element, filter.left) > evaluateN(element, filter.right);
+            case Constants.Evaluation.LESS_THAN_OR_EQUALS:
+                return evaluateN(element, filter.left) <= evaluateN(element, filter.right);
+            case Constants.Evaluation.GREATER_THAN_OR_EQUALS:
+                return evaluateN(element, filter.left) >= evaluateN(element, filter.right);
+            case Constants.Evaluation.AND:
+                return evaluate(element, filter.left) && evaluate(element, filter.right);
+            case Constants.Evaluation.OR:
+                return evaluate(element, filter.left) || evaluate(element, filter.right);
+            case Constants.Evaluation.ADD:
+                const left = evaluate(element, filter.left);
+                const right = evaluate(element, filter.right);
+                if (typeof left !== 'string' || typeof right !== 'string') {
+                    return (+left) + (+right);
+                }
+                return left + right;
+            case Constants.Evaluation.SUBTRACT:
+                return evaluateN(element, filter.left) - evaluateN(element, filter.right);
+            case Constants.Evaluation.MULTIPLY:
+                return evaluateN(element, filter.left) * evaluateN(element, filter.right);
+            case Constants.Evaluation.DIVIDE:
+                return evaluateN(element, filter.left) / evaluateN(element, filter.right);
+            case Constants.Evaluation.MODULO:
+                return evaluateN(element, filter.left) % evaluateN(element, filter.right);
+            default:
+                // The specification is large and we don't support all types of
+                // operators/functions
+                const err = 'Unable to evaluate unknown type: ' + filter.type;
+                log.error(err);
+                throw Error(err);
+        }
+    };
+
+    (function () {
+        // This function resets colors and labels to the original state before
+        // running any further filtering. Edges don't have labels by default and
+        // reducing the parsing overhead can improve performance for some graphs.
```

Comment:
What you mean by 'parsing overhead' here? The parsing has already occurred (on the server-side).

Reply:
That sentence is wrong. I think I had something here previously and moved it elsewhere but did not fix it properly afterwards. I have gotten rid of this line altogether and re-written it properly.